docker-compose.yml

```diff
@@ -1,6 +1,11 @@
 version: '3'
 
 services:
+  test_browser:
+    build:
+      context: .
+      dockerfile: ./Dockerfile-development
```

Comment:
this should be in `docker-compose.override.yml` because it is development-specific rather than common to `dev` and `ci`

Reply:
CI has always used this with the `app` (see just a few lines further down)

Reply:
I'm not saying this shouldn't be changed - but as a separate ticket when we sort out the actual build of the app.